lits.properties

```diff
@@ -1,21 +1,15 @@
 category=Languages
 description=Language Team Integration Tests Support
 devVersion=
-privateVersions=0.5,0.6
+privateVersions=0.6
```

Comment:
Does this even still belong in the Marketplace? Worth checking if the Languages team still uses this.